src/components/FlyoutMenu/FlyoutMenu.jsx

```diff
@@ -0,0 +1,117 @@
+import './_flyout-menu.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SettingsAdjust16 as SettingsAdjust } from '@carbon/icons-react';
+import { Tooltip, Button } from 'carbon-components-react';
+
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const FlyoutMenu = props => {
+  const { children, onApply, onCancel } = props;
+  return (
+    <Tooltip {...props} renderIcon={SettingsAdjust}>
```

Comment:
Not sure how to work this into the current tooltip (you might have to define your own tooltip trigger button) but according to the spec the triggering button should stay in "focus mode" while the flyout is open.
https://github.ibm.com/Watson-IoT/pal-tracking/issues/44#issuecomment-16176683